src/AutoRest.CSharp.V3/Output/Models/Types/EnumType.cs

```diff
@@ -28,7 +28,7 @@ private EnumType(Schema schema, BuildContext context, IEnumerable<ChoiceValue> c
             Declaration = BuilderHelpers.CreateTypeAttributes(
                 schema.CSharpName(),
                 $"{context.DefaultNamespace}.Models",
-                Accessibility.Public,
+                "public",
```

Comment:
Are we no longer using the Accessibility enum?

Reply:
I don't think it was adding anything.